products/llbuildSwift/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ list(APPEND additional_args -DLLBUILD_C_API_VERSION_${LLBUILD_C_API_VERSION} -I
  
 if(APPLE)
   list(APPEND additional_args -target x86_64-apple-macosx10.10)
+else()
+  list(APPEND additional_args -L${FOUNDATION_BUILD_DIR}/Foundation -L${LIBDISPATCH_BUILD_DIR}/src/.libs -I${FOUNDATION_BUILD_DIR}/Foundation -I${FOUNDATION_BUILD_DIR}/Foundation/usr/lib/swift -I${LIBDISPATCH_SOURCE_DIR})
```

Comment:
Maybe we should check if these variables are set and append these arguments if they are. I think we can build the bindings without any additional paths when a toolchain is installed. That would be nice for local development.

Reply:
Done